### ISSUE TITLE
Verify separated /usr support

### ISSUE THREAD

**Issue by adrian15** [rescatux/rescatux#33]
```
lrwxrwxrwx 1 root root 14 Sep 6 17:48 /mnt/test/bin/mount -> /usr/bin/mount
ls: cannot access /mnt/test/sbin/mount: No such file or directory
```

Now we mount all the partitions from fstab thanks to "mount -a" inside the chroot.

This should be resolved but let's try with a distribution that has its /usr partition separated.

Maybe Fedora or opensuse.

**Comment by adrian15** [comment]
0) Bindmount /dev, /proc and /sys
1) Define where you want to mount the parition. e.g. /mnt/rescatux/sda1
2) Read fstab:
/dev/mapper/fedora-root / ext4 defaults 1 1
UUID=6b68 /boot ext4 defaults 1 2
3) Prepend our custom mount point into a temp fstab: /tmp/rescatux-fstab-sda1
/dev/mapper/fedora-root /mnt/rescatux/sda1/ ext4 defaults 1 1
UUID=6b68 /mnt/rescatux/sda1/boot ext4 defaults 1 2
4) mount -a --fstab /tmp/rescatux-fstab-sda1
5) chroot and run programs
6) umount all the filesystems found in /etc/mtab which have: /mnt/rescatux/sda1 on them in reverse order than in mtab file.

**Comment by adrian15** [comment]
Fixed in commit: 1825225d0590286e9579677c5d6e3fdfd3f36fe8 .
